openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -276,23 +277,40 @@ This profile mandates the support for X.509 certificate-based key resolution to
 
 * If the credential has cryptographic holder binding, a KB-JWT, as defined in [@!I-D.ietf-oauth-sd-jwt-vc], MUST always be present when presenting an SD-JWT VC.
 
-# Crypto Suites
+# Crypto Suites {#crypto-suites}
 
 Cryptography is required by the following operations:
 
-- to sign and validate the signature on the Wallet Attestation and its proof of possession
+- to sign and validate the signature on the Wallet or Key Attestation, and their proof of possession (PoP)
+- to sign and validate the signature of the status information of the Verifiable Credential or Wallet Attestation
 - to sign and validate the Issuer's signature on the Verifiable Credential
 - to sign and validate the Holder's signature on the Verifiable Presentation
 - to sign and validate the Verifier's signature on the Presentation Request
 
-Issuers, Holders, and Verifiers MUST support P-256 (secp256r1) as a key type with the ES256 JWT algorithm [@!RFC7518] for the creation and verification of the above signatures.
+All parties MUST, at a minimum, support ECDSA with P-256 and SHA-256 (JOSE algorithm identifier `ES256`; COSE algorithm identifier `-7`, as applicable) for the following:
+
+- Issuers
+  - validating Wallet Attestations (including PoP) when Annex E of [@!OIDF.OID4VCI] is used;
+  - validating Key Attestations when Annex D of [@!OIDF.OID4VCI] is used.
+- Verifiers
+  - validating the signature of the Verifiable Presentation, e.g., KB-JWT of an SD-JWT VC, or `deviceSignature` CBOR structure in case of ISO mdocs. Verifiers are assumed to determine in advance the cryptographic suites supported by the ecosystem, e.g. mDL Issuers/Verifiers implementing ISO mdocs.
```

Comment:
```suggestion
  - validating the signature of the Verifiable Presentation, e.g., KB-JWT of an SD-JWT VC, or `deviceSignature` CBOR structure in case of ISO mdocs. Verifiers are assumed to determine in advance the cryptographic suites supported by the ecosystem, e.g. mDL Issuers/Verifiers implementing ISO mdocs.
  - validating the status information of the Verifiable Credential or Wallet Attestation.
```